src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Loading.java

```diff
@@ -0,0 +1,77 @@
+package com.wikia.webdriver.testcases.globalnavigationbar;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.HomePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.globalnav.GlobalNavigationPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialWikiActivityPageObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Test(groups = {"globalnavigationbar", "Loading"})
+public class Loading extends NewTestTemplate{
+
+  private static final Dimension HUBS_IN_DROPDOWN_RESOLUTION = new Dimension(768, 1024);
+  private static final Dimension HUBS_OUTSIDE_DROPDOWN_RESOLUTION = new Dimension(1024, 1024);
+  private static final List<String> EXPECTED_LINKS_BIG_RESOLUTION =
+      Arrays.asList("Top Communities", "Community Central", "START A WIKIA");
+  private static final List<String> EXPECTED_LINKS_SMALL_RESOLUTION =
+      Arrays.asList("Games", "Movies", "TV", "Top Communities", "Community Central", "START A WIKIA");
+
+  private final static String deWikiName = "de.gta";
+  private static final Dimension HIDE_LOGO_RESOLUTION = new Dimension(1200, 720);
+
+  @Test(groups = {"globalnavigationbar_Loading_001"})
+  public void globalNavigationBarIsFixedOnScroll() {
```

Comment:
why this naming convention? For me it was very convenient when the group was a part of method name.
This way when I was checking logs and I only see method names I could copy the first part of the name and run the test with this group - right now it's not possible without checking the code ;(